### ISSUE TITLE
[fvt]2.13.1 system logs need to return to control node if one case stopped in auto test environment

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#2355]
env:all os
build:latest build
How to reproduce:
```
1.if one case stopped,from control node:
   c910f04x31:/jenkins/log/sles12.1-x86_64/162 # ls -al
total 6016
drwxr-xr-x  2 root root   65536 Dec 27 20:42 .
drwxr-xr-x 53 root root   65536 Dec 27 10:25 ..
-rw-r--r--  1 root root    2363 Dec 27 10:37 cluster.conf
-rw-r--r--  1 root root       0 Dec 27 14:53 failedcases.20161227122007
-rw-r--r--  1 root root   57004 Dec 27 14:53 failedcases.20161227122319
-rw-r--r--  1 root root    7889 Dec 27 14:53 log.sles12.1-x86_64-162
-rw-r--r--  1 root root     565 Dec 27 14:53 mail.sles12.1-x86_64-162
-rw-r--r--  1 root root     218 Dec 27 14:53 mkvm.12496.rec
-rw-r--r--  1 root root     218 Dec 27 14:53 mkvm.12522.rec
-rw-r--r--  1 root root     218 Dec 27 14:53 mkvm.12598.rec
-rw-r--r--  1 root root     218 Dec 27 14:53 mkvm.12624.rec
-rw-r--r--  1 root root    2714 Dec 27 10:38 new_xcattest_installation.log
-rw-r--r--  1 root root    2239 Dec 27 14:53 performance.report.20161227122007
-rw-r--r--  1 root root  128302 Dec 27 14:53 performance.report.20161227122319
.....
-rw-r--r--  1 root root    4264 Dec 27 10:38 sles12.1_x86_64.bundle
-rw-r--r--  1 root root   33062 Dec 27 14:53 xcattest.log.20161227122007
-rw-r--r--  1 root root 3587948 Dec 27 14:53 xcattest.log.20161227122319
c910f04x31:/jenkins/log/sles12.1-x86_64/162 #

2.
 -------------------------> no computes.log,commands.log,cluster.log,messages  if one case stopped
```

**Comment by hu-weihua** [comment]
@junxiawang, I have fixed this issue by PR #50 in ibm github, could you verify it?

One more thing, could you open all issues against auto test framework in ibm internal github, it's a better option. thanks.

**Comment by tingtli** [comment]
@junxiawang  Please open the automation related(xcatjktest, test.sh, configuration files etc) defect under 
https://github.ibm.com/xcat2/jenkins/issues
